### PR TITLE
🚸 socket: hardcode VELO, add USD value

### PR DESCRIPTION
**Pull request by cruzdanilo**

### CODE REVIEW

File: hooks/useSocketAssets.ts [hunk 1]
```diff
@@ -15,46 +16,63 @@ const ETH = {
   icon: '/img/assets/WETH.svg',
   logoURI: '/img/assets/WETH.svg',
   amount: 0,
+  usdAmount: 0,
 } satisfies AssetBalance;
 
+const VELO = {
+  chainId: 10,
+  address: '0x9560e827aF36c94D2Ac33a39bCE1Fe78631088Db',
```
Comment (cruzdanilo) [inline hooks/useSocketAssets.ts:24]: @sebipap get it from the json file

File: hooks/useSocketAssets.ts [hunk 2]
```diff
@@ -15,46 +16,63 @@ const ETH = {
...
+  name: 'Velodrome V2',
```
Comment (cruzdanilo) [inline hooks/useSocketAssets.ts:25]: only _Velodrome_

File: hooks/useSocketAssets.ts (hunk 1 above)
Comment (sebipap) [reply hooks/useSocketAssets.ts:24]: what JSON file? we don't use it anymore.

Comment (cruzdanilo) [reply hooks/useSocketAssets.ts:24]: `@exactly/protocol/deployments/optimism/VELO.json`

File: hooks/useSocketAssets.ts (hunk 2 above)
Comment (cruzdanilo) [reply hooks/useSocketAssets.ts:25]: the price of it is also not working, and we have that price (from socket)

Comment (sebipap) [reply hooks/useSocketAssets.ts:25]: ok, I can fetch it

**Review by jgalat — DISMISSED** [review]
lgtm!

**Pull request merged**